File: good128mb.py
import secrets

# Define the size of the data (1 MB)
size_in_bytes = 1024 * 1024  # 1 MB

# Open the file to write securely generated data
with open('good.dat', 'wb') as f:
    # Write 1 MB of cryptographically secure random bytes
    f.write(secrets.token_bytes(size_in_bytes))

print("Generated 1 MB of cryptographically secure data in good.dat.")

File: good128mb.java
import java.io.FileOutputStream;
import java.io.IOException;
import java.security.SecureRandom;
 
public class GoodDataGenerator {
    public static void main(String[] args) {
        try (FileOutputStream fos = new FileOutputStream("good128java.dat")) {
            SecureRandom secureRandom = new SecureRandom();
            byte[] buffer = new byte[1024 * 1024]; // 1 MB buffer
 
            for (int i = 0; i < 128; i++) { // Write 128 MB
                secureRandom.nextBytes(buffer);
                fos.write(buffer);
            }
            System.out.println("Good data generated: good128java.dat");
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}


